scripts/build.sh

```diff
@@ -343,15 +348,15 @@ RELEASE_NAME=${RELEASE_NAME_MAP[$RELEASE_TYPE]} || abort
 echo -e "Build: RELEASE_TYPE=$RELEASE_NAME"
 
 WSA_ZIP_PATH=$DOWNLOAD_DIR/wsa-$RELEASE_TYPE.zip
-vclibs_PATH=$DOWNLOAD_DIR/Microsoft.VCLibs."$ARCH".14.00.Desktop.appx
-xaml_PATH=$DOWNLOAD_DIR/Microsoft.UI.Xaml_"$ARCH".appx
+vclibs_PATH="$DOWNLOAD_DIR/Microsoft.VCLibs.$ARCH.14.00.Desktop.appx"
+xaml_PATH="$DOWNLOAD_DIR/Microsoft.UI.Xaml_$ARCH.appx"
 MAGISK_ZIP=magisk-$MAGISK_VER.zip
 MAGISK_PATH=$DOWNLOAD_DIR/$MAGISK_ZIP
 KERNEL_VER="5.10.117.2" # TODO: Get from kernel
 KERNELSU_ZIP_NAME=kernelsu-$ARCH-$KERNEL_VER.zip
 KERNELSU_PATH=$DOWNLOAD_DIR/$KERNELSU_ZIP_NAME
-KERNELSU_INFO="$KERNELSU_PATH".info
-if [ "$CUSTOM_MAGISK" ] && [ "$ROOT_SOL" = "magisk" ]; then
+KERNELSU_INFO="$KERNELSU_PATH.info"
+if [ "$CUSTOM_MAGISK" ]; then
```

Comment:
Why you delete the root_sol check bro ? Because if a stupid users turn on KernelSU option and the Custom Magisk argument too, the script will have error

Reply:
Even if Root Solutions does not want Magisk
But we still need to use magiskpolicy to write the rules of GApps into the system
So it is allowed to use custom Magisk at this time

Reply:
Oh, thanks